webpack/production.js

```diff
@@ -69,7 +69,7 @@ export default (webpackConfig) => {
     */
     new WebpackRequireFrom({
       variableName: 'publicUrl',
-      suppressErrors: true, 
+      suppressErrors: true
     }),
     new MiniCssExtractPlugin({
       filename: '[name].[hash:6].css',
```

Comment:
In the new version we have to **replace** hash with **contenthash**, don't we ?

Reply:
It seems like we **can** use contenthash. But hash still works

Reply:
so then it's ok)